### ISSUE TITLE
Theil-Sen Trend Analysis 

### ISSUE THREAD

**Issue by morshedahmed17** [davidcarslaw/openair#378]
### Question

I removed some months from the dataset for the Theil-Sen trend analysis, which I do not want to show in the plot. However, those points still appear in the plots, not exactly where they were before. Is it possible to remove them completely? Also, is it possible to remove the line joining the points?

**Comment by jack-davison** [comment]
Hi,

We'd be able to best help you if you can provide a [reproducible example](https://bookdown.org/david_carslaw/openair/sections/appendices/appendix-gethelp.html).

`TheilSen()` should only plot the data you use as an input, e.g.,:

``` r
library(openair)

alldata <- mydata

TheilSen(alldata, "nox")
#> Taking bootstrap samples. Please wait.
```

![](https://i.imgur.com/7WTn9kw.png)

``` r

lessdata <- selectByDate(mydata, year = 2000:2002)

TheilSen(lessdata, "nox")
#> Taking bootstrap samples. Please wait.
```

![](https://i.imgur.com/hhePXf3.png)

<sup>Created on 2024-02-21 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

More than happy to help if you can provide more info!

Jack

**Comment by morshedahmed17** [comment]
You are right. Just one quick question. 

"If I set 'deaseason = FALSE', will the resulting trendline simply be a regression line?"

**Comment by mooibroekd** [comment]
No, the output will still be a Theil-Sen analysis, but on data that has not been deseasonalized (meaning actual measurements).
